### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by pborreli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,6 @@ experience classes: 'great', 'acceptable' and 'not acceptable'.
 ## Building the CLI
 This will generate a binary executable named `compare_pngs`
 
-_unix based oparting systems_
+_unix based operating systems_
```
Comment (sbussard) [inline README.md]: Good catch, thanks

**Comment by ninokcp3** [comment]
operating systems_

**Comment by pborreli** [comment]
@ninokcp3 sorry ?

**Comment by ninokcp3** [comment]
operating systems_

**Comment by jyrkialakuijala** [comment]
This seems obsoleted now, sorry for not merging it in time

**Pull request closed without merging**